client/plots/cuminc.js

```diff
@@ -675,7 +675,7 @@ function setRenderers(self) {
 				...self.noData.map(chartId => {
 					let chartTitle = chartId
 					const t0 = self.config.term0
-					if (t0.q && t0.q.groupsetting && t0.q.groupsetting.inuse) return { chartId, chartTitle }
+					if (t0.q?.type == 'predefined-groupset' || t0.q?.type == 'custom-groupset') return { chartId, chartTitle }
```

Comment:
todo later/note: this slightly verbose check for a groupsetting q will go away with the tw object refactor